src/main/java/no/nav/folketrygdloven/beregningsgrunnlag/regelmodell/grunnlag/inntekt/Inntektsgrunnlag.java

```diff
@@ -73,10 +73,11 @@ public void leggTilPeriodeinntekt(Periodeinntekt periodeinntekt) {
             .max(Comparator.comparing(Periodeinntekt::getFom));
     }
 
-    public Optional<Periodeinntekt> getOppgittInntektFL() {
+    public List<Periodeinntekt> getOppgittInntektFLIPeriode(Periode periode) {
```

Comment:
Kan man oppgi flere inntektsperioder pr status?

Reply:
Eg antar at vi i juni vil få inn inntektsperioder for både april og mai. Tenkjer i alle fall å lage støtte for det.

Reply:
Vi kan uansett ikkje anta at inntektsperioden oppgitt for frilans gjelder for alle perioder.